packages/website/docs/MultilineBox.md

```diff
@@ -0,0 +1,45 @@
+# Best Practices
+The Multiline box allows users to **add the same sequence of inputs multiple times**.  
+Each sequence appears on a different line, making it easier to consult.  
+
+Do not use the multiline box to regroup inputs that have no connection to one another.   
+Multiline works best when the sequence of inputs is identical on each line.  
+
+Multiline box can combine multiple inputs together to form a more complex input.  
+They can also combine text elements when an input or sequence requires more explanation.  
+
+Only use the drag and drop function if the order of each sequence is important.   
+When doing so, the list must always be presented to the user in the order they have set.
+
+# Labeling
+Keep titles short, preferably **under three words**.  
+
+If the list of inputs is very simple, provide a descriptive title without action verbs.  
+For example, write "Email list" rather than "Provide a list of emails”.
+
+If the sequence of inputs is more complex, consider using a title with a trigger verb to clearly explain that the outcome will apply or consider the entire list.  
+For example, write “Show content that matches" instead of "Filter rules”. 
+
+When using a dialog approach, always consider other inputs in the form so that it is cohesive with other inputs.
```

Comment:
```suggestion
You may choose to use a dialog approach with the labels. For example, you could have: "Among all items matching... [condition1] Show only the items with... [condition2]". In such case, consider the other inputs in the form so that your approach is coherent with all inputs.
```